robottelo/cli/org.py

```diff
@@ -168,15 +168,3 @@ def remove_user(cls, options=None):
         """Removes an user from an org"""
         cls.command_sub = 'remove-user'
         return cls.execute(cls._construct_command(options))
-
-    @classmethod
-    def set_parameter(cls, options=None):
-        """Create or update parameter for an organization."""
-        cls.command_sub = 'set-parameter'
-        return cls.execute(cls._construct_command(options))
-
-    @classmethod
-    def delete_parameter(cls, options=None):
```

Comment:
Search on the code shows that curently `set_parameter` implemented only in `Base` class and `Org`. It was added to `Org` by me in PR #4126 only because I've missed that there is base implementation. That's why I've decided to remove this excess code in this PR.